Empty databases/openIMIS_ONLINE.sql

```diff
@@ -3613,6 +3623,21 @@ GO
 SET QUOTED_IDENTIFIER ON
 GO
 
+
+CREATE TABLE tblBulkControlNumbers 
+(
+	Id INT IDENTITY(1, 1) CONSTRAINT PK_tblBulkControlNumbers PRIMARY KEY,
+	BillId INT UNIQUE CONSTRAINT UQ_tblBulkControlNumbers_BillId NOT NULL,
```

Comment:
why didn't you put an FK to Tblpayment.PaymentId ?

Reply:
Because this is independent of that payment. This control numbers has no reference to any payment yet.